slp_drawio/slp_drawio/parse/tranformers/parent_calculator_transformer.py

```diff
@@ -0,0 +1,24 @@
+from otm.otm.entity.parent_type import ParentType
+from slp_drawio.slp_drawio.objects.diagram_objects import DiagramComponent, DiagramTrustZone
+from slp_drawio.slp_drawio.parse.transformer.transformer import Transformer
+
+PARENT_TYPES = {
+    DiagramComponent: ParentType.COMPONENT,
+    DiagramTrustZone: ParentType.TRUST_ZONE
+}
+
+
+def get_parent_type(element) -> ParentType:
+    return PARENT_TYPES[element.__class__]
+
+
+class ParentCalculatorTransformer(Transformer):
```

Comment:
Parent calculator should be used at DrawioParser.build_otm bellow  the line # TODO Implement and call Transformers here

Reply:
I'm not sure if this is the right task to do that, but also done anyway